config/const.go

```diff
@@ -55,6 +55,23 @@ var DefaultGCRRegistries = [...]string{
 	"marketplace.gcr.io",
 }
 
+// DefaultARRegistries contains the list of default registries for Artifact
+// Registry.  If the --include-artifact-registry flag is supplied then these
+// are added in addition to the GCR Registries.
+var DefaultARRegistries = [...]string{
```

Comment:
If this were a separate file (json or something), it might be easier to automate updates to this list, FWIW.

Reply:
Could pull these var declarations into a dedicated file(s) in the same package, which themselves could be easily created from a list of hostnames. How often are y'all thinking of updating this list? :p

Reply:
We'll come back to this in a few months when we add more regions, I think I'll leave the refactoring until then.